examples/java8/src/main/java/org/apache/beam/examples/complete/game/GameStats.java

```diff
@@ -193,6 +193,8 @@ public void processElement(ProcessContext c) {
   /**
    * Create a map of information that describes how to write pipeline output to BigQuery. This map
    * is used to write information about team score sums.
+   *
+   * @return The map of fields to write on big query.
```

Comment:
We should really disable whatever requirement is for this. The statement is often redundant with the basic javadoc and just adds burden. It is useful for inheritance situations because it is separate from `{@inheritDoc}` but otherwise I don't think it matters much.